js/App.js

```diff
@@ -125,11 +127,33 @@ App.prototype.setUnreadChatMessageCount = function(count) {
   }
 };
 
-App.prototype.intlNumFormat = function(numberToFormat, maxDigits){
-  maxDigits = maxDigits || 8; //default to show down to the satoshi (.00000001)
+App.prototype.intlNumFormat = function(numberToFormat, maxDigits) {
+  if (maxDigits === undefined) { // Allow maxDigits = 0
+    maxDigits = 8; //default to show down to the satoshi (.00000001)
+  }
   return new Intl.NumberFormat(window.lang, {maximumFractionDigits: maxDigits}).format(numberToFormat);
 };
 
+/**
+ * Format a bitcoin amount in the user's locale.
+ *
+ * @param {Number} amount - Bitcoin (BTC) amount.
+ * @param {Number} [maxDigits=8] - Maximum number of fraction digits to display.
+ * @returns {String} localised amount ending with ' BTC'
+ *
+ * @see intlNumFormat
+ */
+App.prototype.formatBitcoin = function(amount, maxDigits) {
+  if (_app.bitcoinUnit === 'mBTC') {
```

Comment:
hmm... good catch. Yeah what `this` ends up referring to is the particular context that is passed into the template. I'm not really sure how that's happening. I need to think about that.

Anyhow, for the time being to work around, I would just adjust the template helper code like so:

```
module.exports = {
  formatBitcoin: function() {
    return app.formatBitcoin.apply(app, arguments);
  }
}
```